DTS-STN/Alpha-Site
# Updated project and Dockerfile to use yarn

# Description

[SCL-592- Refactor Alpha site Dockerfile](https://jira-dev.bdm-dev.dts-stn.com/browse/SCL-592)

This PR removes `package-lock.json` and replaces it with `yarn.lock`. The Dockerfile has been updated to use yarn instead of npm and has also been re-arranged slightly based on Next.js' Dockerfile example.

I updated the GitHub actions to reflect the above change and also updated Storybook to use PostCSS 8 in order for it to properly build with Tailwind 3.

## Test Instructions

1. Pull in branch
2. Type `yarn dev` to ensure it works as expected
3. Type `docker build -t alpha-site:latest .`
4. Type `docker run -p 3000:3000 alpha-site:latest`
5. Navigate through site to ensure everything works as expected

## Checklist


## Product and Sprint Backlog

[Jira](https://jira-dev.bdm-dev.dts-stn.com/secure/RapidBoard.jspa?rapidView=96&projectKey=SCL)
